tmcprototype/DishLeafNode/DishLeafNode/DishLeafNode.py

```diff
@@ -826,8 +826,8 @@ def Track(self, argin):
             dec_value = (jsonArgument["pointing"]["target"]["dec"])
             radec_value = 'radec' + ',' + str(ra_value) + ',' + str(dec_value)
             self.event_track_time.clear()
-            self.tracking_time_thread1 = threading.Thread(None, self.tracking_time_thread, CONST.THREAD_TRACK)
-            self.tracking_time_thread1.start()
+            # self.tracking_time_thread1 = threading.Thread(None, self.tracking_time_thread, CONST.THREAD_TRACK)
```

Comment:
Commented code can be removed.

Reply:
Added TODO tag. It may be required later.